cli/script/command-parser.ts

```diff
@@ -316,7 +316,7 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
         isValidCommandCategory = true;
         isValidCommand = true;
         yargs.usage(USAGE_PREFIX + " login [options]")
-            .demand(/*count*/ 1, /*max*/ 2)  // Require one non-optional and one optional argument.
+            .demand(/*count*/ 0, /*max*/ 0)  
```

Comment:
While you're at it, could you please leave a (short) comment explaining that, here and in the other places?